src/processors/processor.ts

```diff
@@ -123,6 +127,7 @@ export default class Processor {
           const child = item.children[0];
           const name = child.name ?? child.value.replace(/'|"/g, '');
           const varName = child.type === 'String' ? name.replace(/\./, '-') : name;
+          if (!this.ast.css) throw new Error('ast.css is not defined.');
```

Comment:
Hack: Needed to do this to pass linting since the type of `Ast.css` changed to be nullable.